docs/intro/index.rst

```diff
@@ -1,23 +1,37 @@
 .. _guide:
 
-***************
-Getting Started
-***************
+**********************
+The ``sunpy`` tutorial
```

Comment:
I really don't like the highlighting effect of using verbatim blocks in titles (generally).

This:
![title](https://user-images.githubusercontent.com/1391051/203363142-6335777d-4224-46b3-8690-5af81cd8d8c4.png)

just looks ugly to me.

```suggestion
The sunpy tutorial
```

Reply:
👍 agreed